src/reynir_correct/pattern.py

```diff
@@ -174,12 +176,19 @@ def wrong_preposition_af(self, match: SimpleTree) -> None:
         # Find the attached prepositional phrase
         pp = match.first_match('P > { "af" }')
         # Calculate the start and end token indices, spanning both phrases
-        start, end = min(vp.span[0], pp.span[0]), max(vp.span[1], pp.span[1])
-        text = "'{0} af' á sennilega að vera '{0} að'".format(vp.tidy_text)
-        detail = (
-            "Sögnin '{0}' tekur yfirleitt með sér "
-            "forsetninguna 'að', ekki 'af'.".format(vp.tidy_text)
-        )
+        try:
+            start, end = min(vp.span[0], pp.span[0]), max(vp.span[1], pp.span[1])
+        except AttributeError:
+            start, end = match.span
+        if vp != None:
```

Comment:
Canonical Python is: `if vp is not None:`

Reply:
Fixed!